output_template/python/usb_camera_demo.py

```diff
@@ -50,34 +46,25 @@
     label_to_color_image,
     visualize_keypoint_detection,
 )
+from lmnet.pre_processor import resize
 
 
 nn = None
 pre_process = None
 post_process = None
 
 
-class MyTime:
-    def __init__(self, function_name):
-        self.start_time = time.time()
-        self.function_name = function_name
-
-    def show(self):
-        print("TIME: ", self.function_name, time.time() - self.start_time)
-
-
 def init_camera(camera_width, camera_height):
     if hasattr(cv2, 'cv'):
         vc = cv2.VideoCapture(0)
         vc.set(cv2.cv.CV_CAP_PROP_FRAME_WIDTH, camera_width)
         vc.set(cv2.cv.CV_CAP_PROP_FRAME_HEIGHT, camera_height)
-        vc.set(cv2.cv.CV_CAP_PROP_FPS, 10)
-
+        vc.set(cv2.cv.CV_CAP_PROP_FPS, 60)
     else:
-        vc = cv2.VideoCapture(1)
+        vc = cv2.VideoCapture(0)
```

Comment:
This number is "camera id", starts from 0.
If a machine has multiple cameras, camera id will be 0, 1, 2, ... .
But it is hard to determine which number is desired for demo.
So I set this number to 0, it is minimum number to be accepted and it may be desired for 
 many environments.

Reply:
I see.